### PR TITLE
feat(activity-button):create

### PR DESCRIPTION
**Pull request by ArthusLiang**
create

updates

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: angular/src/lib/activity-button/activity-button.component.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+export type ActivityButtonSize = 56 | 68 | 84;
+export type ActivityButtonType = 'chat' | 'camera' | 'contact-card' | 'meetings' | 'whiteboard' | 'files' | 'share-screen' | 'tasks' | '';
+
+@Component({
+  selector: 'md-activity-button',
+  template: `
+  <button
+    [ngClass]='btnNgClass'
```
Comment (mathsusan) [inline angular/src/lib/activity-button/activity-button.component.ts]: In another PR it was mentioned that we don't abbreviate button in our new libraries

Comment (bfbiggs) [reply angular/src/lib/activity-button/activity-button.component.ts]: Let's discuss this during standup.  We've recently allowed button to be abbreviated as a prop.

File: angular/src/docs/app/playground/playground.component.ts [hunk 2]
```diff
@@ -11,9 +11,9 @@ import { Component, OnInit } from '@angular/core';
   `,
 })
 export class PlaygroundComponent implements OnInit {
-  ngOnInit() {}
+  ngOnInit() { }
 
-  onSelect (event) {
+  onSelect(event) {
```
[the remark this answers is not in the document]
Comment (ArthusLiang) [reply angular/src/docs/app/playground/playground.component.ts]: Sure

File: angular/src/lib/activity-button/activity-button.component.ts (hunk 1 above)
Comment (ArthusLiang) [reply angular/src/lib/activity-button/activity-button.component.ts]: I can change the name btnNgClass to buttonNgClass. Thanks for keep update.

**Pull request merged**